[2024-10-26T21:02:06.468+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-26T21:02:06.485+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_scheduler.hello manual__2024-10-26T21:02:02.738261+00:00 [queued]>
[2024-10-26T21:02:06.493+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_scheduler.hello manual__2024-10-26T21:02:02.738261+00:00 [queued]>
[2024-10-26T21:02:06.495+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-26T21:02:06.508+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): hello> on 2024-10-26 21:02:02.738261+00:00
[2024-10-26T21:02:06.517+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_scheduler', 'hello', 'manual__2024-10-26T21:02:02.738261+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/extractor.py', '--cfg-path', '/tmp/tmps3f8via3']
[2024-10-26T21:02:06.516+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=671) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-26T21:02:06.518+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask hello
[2024-10-26T21:02:06.518+0000] {standard_task_runner.py:72} INFO - Started process 673 to run task
[2024-10-26T21:02:06.637+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_scheduler.hello manual__2024-10-26T21:02:02.738261+00:00 [running]> on host c94222e110db
[2024-10-26T21:02:06.716+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_scheduler' AIRFLOW_CTX_TASK_ID='hello' AIRFLOW_CTX_EXECUTION_DATE='2024-10-26T21:02:02.738261+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-26T21:02:02.738261+00:00'
[2024-10-26T21:02:06.718+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-26T21:02:06.718+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-26T21:02:06.719+0000] {logging_mixin.py:190} INFO - Current task name:hello state:running start_date:2024-10-26 21:02:06.486604+00:00
[2024-10-26T21:02:06.720+0000] {logging_mixin.py:190} INFO - Dag name:dag_scheduler and current dag run status:running
[2024-10-26T21:02:06.720+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-26T21:02:06.722+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-26T21:02:06.723+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world']
[2024-10-26T21:02:06.732+0000] {subprocess.py:86} INFO - Output:
[2024-10-26T21:02:06.733+0000] {subprocess.py:93} INFO - hello world
[2024-10-26T21:02:06.734+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-26T21:02:06.758+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-26T21:02:06.759+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_scheduler, task_id=hello, run_id=manual__2024-10-26T21:02:02.738261+00:00, execution_date=20241026T210202, start_date=20241026T210206, end_date=20241026T210206
[2024-10-26T21:02:06.773+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-26T21:02:06.773+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-26T21:02:06.774+0000] {logging_mixin.py:190} INFO - Dag name:dag_scheduler queued_at:2024-10-26 21:02:02.772220+00:00
[2024-10-26T21:02:06.774+0000] {logging_mixin.py:190} INFO - Task hostname:c94222e110db operator:BashOperator
[2024-10-26T21:02:06.823+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-26T21:02:06.838+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-26T21:02:06.843+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
